index.html

```diff
@@ -87,6 +91,8 @@
             <br>
         </div>
         <br>
+        <div class="plyr"><audio></audio></div>
```

Comment:
I spent about an hour trying to figure out why Plyr was throwing `Uncaught TypeError`. Finally I remembered that I didn't change those `<div>`'s to `<audio>` or `<video>`. I don't know if I should feel bad because I spent so much time figuring this out or good because I'm sure `<div>`'s don't work properly in there in our case. It doesn't matter that much but Plyr was throwing this error because `buttons` elements' references weren't there. It was trying to access `plyr.buttons.play` but `buttons` was undefined (Of course I'm talking about Plyr's code, not ours).